### PR TITLE
V0.5

### PR DESCRIPTION
**Pull request by DarrenWestwood**
Altcoin integration, refunds, email, qrcode all working.

### CODE REVIEW

**Comment by blockonomics** [comment]
@jusasiiv  please check basic code quality and functionality. We can do detailed testing later in a spreadsheet

**Review by jusasiiv — COMMENTED** [review]
Magento related code looks good and clean, follows existing patterns

No need to include both _cryptofont.css_ and _cryptofont.min.css_ 

Is _view/frontend/web/css/material/README.md_  necessary to include? Also is this needed:  _view/frontend/web/css/material/codepoints_ ?

_view/frontend/web/js/createQr.js_ should be renamed or code separated. Here we use lots of `elseifs`, is there an issue on re-using existing/most of existing flyp.me JS? This JS feels hard to follow

**Comment by DarrenWestwood** [comment]
> No need to include both cryptofont.css and cryptofont.min.css
> 
> Is view/frontend/web/css/material/README.md necessary to include? Also is this needed: view/frontend/web/css/material/codepoints ?

Fixed in 503859c - Only load cryptofont.min.css and icons.css

> view/frontend/web/js/createQr.js should be renamed or code separated.

Fixed in 772ca1e - Rename createQr.js to app.js

> This JS feels hard to follow

Fixed in ab17ee9 and 62a277d - Improve altcoin integration to match the existing JS

**Review by jusasiiv — APPROVED** [review]
> @jusasiiv please check basic code quality and functionality. We can do detailed testing later in a spreadsheet

Code changes look good. Tested basic functionality and verified that everything is working

Screenshot_2019-07-30 Screenshot
Screenshot_2019-07-30 Screenshot(1)
Screenshot_2019-07-30 Screenshot(2)
Screenshot_2019-07-30 Screenshot(3)
Screenshot_2019-07-30 Screenshot(4)

**Comment by blockonomics** [comment]
Please upload release on github and marketplace
